nexus_workcell_orchestrator/src/workcell_orchestrator.cpp

```diff
@@ -214,7 +214,7 @@ auto WorkcellOrchestrator::on_activate(
   RCLCPP_INFO(this->get_logger(), "Workcell activated");
   this->_bt_timer = this->create_wall_timer(BT_TICK_RATE, [this]()
       {
-        this->_tick_all_bts();
+        this->_job_mgr->tick();
```

Comment:
Should we check whether `_job_mgr.has_value()` before ticking? And same within on_deactive, on_cleanup, etc.

Reply:
Using `this->_job_mgr.value().tick();` instead, if there is no value it will throw an exception.